pkg/kernelsyms/kernel_symbols.go

```diff
@@ -0,0 +1,86 @@
+package kernelsyms
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type KernelSymbolTable struct {
+	symbolMap map[string]KernelSymbol
+}
+
+type KernelSymbol struct {
+	Name    string
+	Type    string
+	Address uint64
+	Owner   string
+}
+
+type HookedSyscallData struct {
+	SyscallName string
+	ModuleOwner string
+}
```

Comment:
I think `HookedSyscallData` isn't  should be there but the `KernelSymbolTable` and `KernelSymbol` should be because the package  give's object of all the symbols  in the kernel